tests/integration/release/test_release.py

```diff
@@ -14,131 +15,114 @@
-    assert (noised_data.dtypes == original_data.dtypes).all()
+    for noise_type in NOISE_TYPES:
+        prenoised_dataframes = [dataset.data.copy() for dataset in datasets]
+        if isinstance(noise_type, RowNoiseType):
+            if config.has_noise_type(dataset_schema.name, noise_type.name):
+                [noise_type(dataset, config) for dataset in datasets]
+                test_function = ROW_TEST_FUNCTIONS[noise_type.name]
+                test_function(
+                    prenoised_dataframes, datasets, config, full_dataset_name, fuzzy_checker
+                )
 
 
-@pytest.mark.parametrize("expected_noise", ["default", 0.03])
-def test_do_not_respond(
-    expected_noise: str | float,
+def test_column_noising(
     unnoised_dataset: Dataset,
+    noised_data: pd.DataFrame,
+    config: dict[str, Any],
     dataset_name: str,
     fuzzy_checker: FuzzyChecker,
 ) -> None:
-    """Tests that do_not_respond noising is being applied at the expected proportion."""
-    # do_not_respond only applies to survey data
-    if dataset_name not in [
-        DATASET_SCHEMAS.acs.name,
-        DATASET_SCHEMAS.census.name,
-        DATASET_SCHEMAS.cps.name,
-    ]:
-        return
-
-    original_data = unnoised_dataset.data
-    config_dict = get_single_noise_type_config(dataset_name, NOISE_TYPES.do_not_respond.name)
-
-    if expected_noise != "default":
-        config_dict[dataset_name][Keys.ROW_NOISE][NOISE_TYPES.do_not_respond.name][
-            Keys.ROW_PROBABILITY
-        ] = expected_noise
-        config = NoiseConfiguration(LayeredConfigTree(config_dict))
-    else:
-        config = NoiseConfiguration(LayeredConfigTree(config_dict))
-        # updating expected_noise from 'default' to actual default value
-        expected_noise = config.get_row_probability(
-            dataset_name, NOISE_TYPES.do_not_respond.name
-        )
-    assert isinstance(expected_noise, float)
-
-    dataset_schema = DATASET_SCHEMAS.get_dataset_schema(dataset_name)
-    dataset = Dataset(dataset_schema, original_data, SEED)
-    NOISE_TYPES.do_not_respond(dataset, config)
-    noised_data = dataset.data
-
-    # Check ACS and CPS data is scaled properly due to oversampling
-    if dataset_name is not DATASET_SCHEMAS.census.name:  # is acs or cps
-        expected_noise = 0.5 + expected_noise / 2
+    """Tests that columns are noised as expected"""
+    check_noised, check_original, shared_idx = _get_common_datasets(
+        unnoised_dataset, noised_data
+    )
 
-    # Test that noising affects expected proportion with expected types
-    fuzzy_checker.fuzzy_assert_proportion(
-        name="test_do_not_respond",
-        observed_numerator=len(original_data) - len(noised_data),
-        observed_denominator=len(original_data),
-        # 3% uncertainty on either side
-        target_proportion=(expected_noise * 0.97, expected_noise * 1.03),
-        name_additional=f"noised_data",
+    run_column_noising_tests(
```

Comment:
Why is there column noising stuff in this row-noising PR? Should I be reviewing all this?

Reply:
No you can ignore this